apps/mull-ui-e2e/src/support/commands.ts

```diff
@@ -26,6 +27,18 @@ Cypress.Commands.add('mockRefreshRequest', (userId = 1) => {
     },
   });
 });
+
+Cypress.Commands.add('interceptGraphQLReqs', () => {
+  return cy.intercept('POST', 'localhost:3333/graphql', (req) => {
+    if (req.body.operationName.includes('User')) {
+      req.alias = 'User';
+    }
+    if (req.body.operationName.includes('Login')) {
+      req.alias = 'Login';
+    }
+  });
+});
+
```

Comment:
This tells Cypress to intercept POST requests to our GQL endpoint. In this case, I am checking to see when the User query and Login mutation is called. Then, in the E2E test, I am telling cypress to not advance in the test until a request with `@User` or `@Login` has been made. Only then will the E2E test advance. This is an alternative to `.wait(int)` which is apparently an anti-pattern, and this is the correct way to do a `.wait(alias)` on Cypress. IMO, this helps make the test non-flaky (it was super flaky before I added the intercepts). This could be something to do for some of our flaky E2Es as well.

I'll make sure to add a small explanation of this!

Reply:
Refs: https://docs.cypress.io/api/commands/wait.html#Time